[docs] In regards of filter not detected in some relationships (#252)
OpenCTI-Platform/docs: docs/reference/filters.md

};
```

![RegardingOf filter](./assets/filters-regardingOf.png)

!!! warning "This filter may exclude some results for technical reasons"

    This filter is based on denormalized information for relationships.
    In a given entity, for each of the relationships in which it is involved, we store the relationship type and the id of the other entity involved.

>     But for performance reasons, this denormalized information is not stored anymore since OpenCTI 6.5 for some relationships implicating high data volumes.

Reviewer: ```suggestion
    For performance reasons, this denormalized information is not stored for some relationships involving high data volumes.
```

};
```

![RegardingOf filter](./assets/filters-regardingOf.png)

!!! warning "This filter may exclude some results for technical reasons"

    This filter is based on denormalized information for relationships.
    In a given entity, for each of the relationships in which it is involved, we store the relationship type and the id of the other entity involved.

    For performance reasons, this denormalized information is not stored for some relationships involving high data volumes.

    Thus, the ``regardingOf`` filter does not detect the following relationships:
    - the relationships of type ``related_to`` with an ``Observable`` as source type,

        Example: given the relationship "APT41 (intrusion set) related to Winscp.rnd (file observable)" present in the platform
        APT41 would not be returned when using filter "In regards of" related to Winscp.rnd

    - the relationships of type ``located_at`` with an ``Ipv4/Ipv6 Address`` or a ``City`` as source type, and a ``Region`` or ``Country`` as target type,

>         Example of relationship: Paris located at France

Reviewer: ```suggestion
        Example of relationship: given the relationship "IP 1.1.1.1 (observable) located in Paris (City)"
        "The IP address would not be returned when using filter "In regard of" located at Paris
```

Author: This example is not correct since the target type is a city (and not a Region or Country)

    - the relationships of type ``targets`` with a ``Region``, ``Country`` or ``Sector`` as target type.

>         Example of relationship: MalwareX targets Asia

Reviewer: ```suggestion
        Example: given the relationship "0mega (Malware) targets Energy (Sector)" present in the platform
        Energy would not be returned when using filter "In regard of" targets 0mega
```

    This filter is based on denormalized information for relationships.
    In a given entity, for each of the relationships in which it is involved, we store the relationship type and the id of the other entity involved.

    For performance reasons, this denormalized information is not stored for some relationships involving high data volumes.

    Thus, the ``regardingOf`` filter does not detect the following relationships:
    - the relationships of type ``related_to`` with an ``Observable`` as source type,

        Example: given the relationship "APT41 (intrusion set) related to Winscp.rnd (file observable)" present in the platform
        APT41 would not be returned when using filter "In regards of" related to Winscp.rnd

    - the relationships of type ``located_at`` with an ``Ipv4/Ipv6 Address`` or a ``City`` as source type, and a ``Region`` or ``Country`` as target type,

        Example of relationship: Paris located at France

    - the relationships of type ``targets`` with a ``Region``, ``Country`` or ``Sector`` as target type.

        Example: given the relationship "0mega (Malware) targets Energy (Sector)" present in the platform

>         Energy would not be returned when using filter "In regard of" targets 0mega

Reviewer: ```suggestion
        Energy would not be returned when using filter "In regards of" targets 0mega
        0mega will be returned when using filter "In regards of" targets Energy
```

};
```

![RegardingOf filter](./assets/filters-regardingOf.png)

!!! warning "This filter may exclude some results for technical reasons"

    This filter is based on denormalized information for relationships.
    In a given entity, for each of the relationships in which it is involved, we store the relationship type and the id of the other entity involved.

    For performance reasons, this denormalized information is not stored for some relationships involving high data volumes.

    Thus, the ``regardingOf`` filter does not detect the following relationships:
    - the relationships of type ``related_to`` with an ``Observable`` as source type,

>         Example: given the relationship "APT41 (intrusion set) related to Winscp.rnd (file observable)" present in the platform
>         APT41 would not be returned when using filter "In regards of" related to Winscp.rnd

Reviewer: ```suggestion
        Example: given the relationship "Winscp.rnd (file observable) related to APT41 (intrusion set)" present in the platform
        APT41 would not be returned when using filter "In regards of" related to Winscp.rnd
        Winscp.rnd would be returned when using filter "In regards of" related to APT41
```

};
```

![RegardingOf filter](./assets/filters-regardingOf.png)

!!! warning "This filter may exclude some results for technical reasons"

    This filter is based on denormalized information for relationships.

>     In a given entity, for each of the relationships in which it is involved, we store the relationship type and the id of the other entity involved.

Reviewer: ```suggestion
    In a given entity, we directly store the id of the entities with which the entity has a relationship. This allows fast queries involving relationship filters.
    
    However, this approach might lead to ever-growing entities in database, with hundreds of thousands of ids stored in a given entity. Take for example a very active Intrusion Set, with tons of observables related to it. The corresponding  denormalization data in the Intrusion Set would represent a significant overhead, leading to dramatical performance drop during some operations manipulating this object.
```

>     For performance reasons, this denormalized information is not stored for some relationships involving high data volumes.

Reviewer: ```suggestion
    For performance reasons, this denormalized information is not stored on all sides for some relationships involving high data volumes.
```

>     For performance reasons, this denormalized information is not stored in the source entity for some relationships involving high data volumes.

Reviewer: ```suggestion
    For performance reasons, this denormalized information is thus not stored in the source entity for some relationships involving high data volumes.
```

};
```

![RegardingOf filter](./assets/filters-regardingOf.png)

!!! warning "This filter may exclude some results for technical reasons"

    This filter is based on denormalized information for relationships.
    In a given entity, we directly store the id of the entities with which the entity has a relationship. This allows fast queries involving relationship filters.
    
    However, this approach might lead to ever-growing entities in database, with hundreds of thousands of ids stored in a given entity. 

>     Take for example a very active Intrusion Set, with tons of observables related to it. 

Reviewer: ```suggestion
    Take for example a very active Intrusion Set, with thousands of observables related to it. 
```
